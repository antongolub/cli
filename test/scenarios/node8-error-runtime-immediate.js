'use strict'
require('../..')(main)

async function main () {
  console.log('started')
  main.test.example = 'will explode'
  console.log('finished')
}
