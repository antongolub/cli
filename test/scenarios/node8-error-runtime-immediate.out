started
TypeError: Cannot set property 'example' of undefined
    at main (scenarios/node8-error-runtime-immediate.js:)
1
